dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -98,7 +98,7 @@ def request
 
   test "get video choices" do
     choices_cached = video_key_choices
-    assert_equal(choices_cached.count, Video.count)
+    assert_equal(choices_cached.count, Video.where(locale: 'en-US').count)
```

Comment:
Why is the Video count specific to US locale?

Reply:
Right now (and for the foreseeable future) all dubbed videos must have an English counterpart, so the set of unique keys is equal to the set of keys for English videos. 

I couldn't think of a great way to test this without just duplicating the logic in the function. This is the best I came up with.